cpan/Digest-MD5/MD5.pm

```diff
@@ -258,12 +258,12 @@ The above example would print out the message:
 The same checksum can also be calculated in OO style:
 
     use Digest::MD5;
-    
```

Comment:
This change is wrong: removing this whitespace will cut the verbatim block into two